README.md

```diff
@@ -223,19 +241,30 @@ Note that you can use MySQL Operators with MariaDb database if compatible.
 	- Recursively sorts keys and removes spaces, allowing comparison of json documents for equality.
 
 ### PostgreSQL 9.3+ JSON operators
-Basic support for JSON operators is implemented. This works even with `Doctrine\DBAL` v2.5. [Official documentation of JSON operators](https://www.postgresql.org/docs/9.3/static/functions-json.html).
-
+Basic support for JSON operators is implemented. This works even with `Doctrine\DBAL` v2.5. [Official documentation of JSON operators](https://www.postgresql.org/docs/9.5/functions-json.html).
+
+* **JSONB_CONTAINS(jsonb, jsonb)**
+	- expands to `jsonb @> jsonb`
+* **JSONB_EXISTS(jsonb, text)**
+	- executed as `JSONB_EXISTS(jsonb, text)`, equivalent to `jsonb ? text`
+* **JSONB_EXISTS_ALL(jsonb, array)**
+	- executed as `JSONB_EXISTS_ALL(jsonb, array)`, equivalent to `jsonb ?& array`
+* **JSONB_EXISTS_ANY(jsonb, array)**
+	- executed as `JSONB_EXISTS_ANY(jsonb, array)`, equivalent to `jsonb ?| array`
+* **JSONB_IS_CONTAINED(jsonb, jsonb)**
+	- expands to `jsonb <@ jsonb`
+* **JSONB_INSERT**
+	- executed as is
+* **JSON_EXTRACT_PATH**
+	- executed as is
```

Comment:
it seems the library provides 2 of the many additional functions listed. i guess those are the ones somebody needed... enough like this or should i add some note?